src/Generation/GapicClientGenerator.php

```diff
@@ -443,7 +443,7 @@ private function getClientDefaults(): PhpClassMember
 
         // TODO: Consolidate setting all the known array values together.
         // We do this here to maintain the existing sensible ordering.
-        if ($this->serviceDetails->transportType === Transport::GRPC_REST) {
+        if ($this->serviceDetails->transportType !== Transport::REST) {
```

Comment:
Ahh I see it now... it's because we need to have this same change in `src/Generation/GapicClientV2Generator.php`, and we want the changes below in `src/Generation/GapicClientGenerator.php`. Either that or we need to throw an error for V1 clients. I am going to look into this!

Reply:
Okay we can fix this by adding the following to `php_gapic.bzl`:

```bzl
if transport == "grpc" and migration_mode != "NEW_SURFACE_ONLY":
    fail("Error: 'grpc' transport is only supported with 'NEW_SURFACE_ONLY' migration mode")
```

Reply:
See https://github.com/googleapis/gapic-generator-php/pull/708/files - I show how to make this happen